x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/config_panel_wrapper.tsx

```diff
@@ -222,33 +222,53 @@ function LayerPanel(
         render={activeVisualization.renderLayerConfigPanel}
         nativeProps={layerConfigProps}
       />
+
+      <EuiSpacer size="s" />
+
+      <EuiFlexItem>
```

Comment:
You'll need a `<EuiFlexGroup justifyContent="center">` surrounding this flex item. As well as `grow=false` on the flex item.